src/test/java/io/confluent/connect/elasticsearch/integration/ElasticsearchContainer.java

```diff
@@ -191,7 +192,7 @@ protected void configure() {
               .forStatusCodeMatching(status -> status == HTTP_OK || status == HTTP_UNAUTHORIZED)
               .withStartupTimeout(Duration.ofMinutes(2))
       );
-      image = new RemoteDockerImage(imageName);
+      image = new RemoteDockerImage(DockerImageName.parse(imageName));
```

Comment:
Is moving from a TestContainer approach to using https://confluentinc.atlassian.net/wiki/spaces/CONNECT/pages/1549697235/Configure+Connectors+Repositories+to+run+integration-tests+on+Jenkins not in the scope of this PR?

It would involve some work to rewrite these tests to hit an actual ES instance, and I'm wondering why add more testing with the Container approach when there is a plan to migrate

Reply:
these ITs are being run in Jenkins right now, on top of that, TestContainers just brings up a docker image of Elasticsearch so these tests are actually running against actual ES instances and even lets us configure the ES instance to however we like programatically (i.e general ssl certs and test those) which we wouldnt be able to do with an "actual" ES instance easily

Reply:
Ah my bad. I thought TestContainers being run on Jenkins had some n/w problems

Reply:
yeah testcontainers is actually a really nifty thing :)

Reply:
So are we running only SecurityIT per this comment? https://github.com/confluentinc/kafka-connect-elasticsearch/blob/66fcdf2594297ebafb5e84190d2d70e61db6f9a5/pom.xml#L334

Reply:
hmm good catch, thats why my test is failing, I will have to disable it